src/server/services/about/views/service.njk

```diff
@@ -54,8 +57,17 @@
   }) }}
 {% endblock %}
 
+
 {% block tabContent %}
 
+  {% for vanityUrl in vanityUrls %}
+    {% for url in vanityUrl.urls | selectattr("shuttered") %}
```

Comment:
Can you wrap this in

```
<div class="govuk-!-margin-bottom-4">
...
</div>
```

To give it a little breathing space at the bottom

Reply:
![image](https://github.com/user-attachments/assets/1d0f3487-a09d-44d1-b310-0f7384ebb5aa)